qiskit/transpiler/passmanager.py

```diff
@@ -33,49 +38,102 @@
 class PassManager(BasePassManager):
     """Manager for a set of Passes and their scheduling during transpilation."""
 
-    PASS_RUNNER = RunningPassManager
-
     def __init__(
         self,
-        passes: PassSequence | None = None,
+        passes: Task | list[Task] = None,
         max_iteration: int = 1000,
     ):
-        """Initialize an empty `PassManager` object (with no passes scheduled).
+        """Initialize an empty pass manager object.
 
         Args:
-            passes: A pass set (as defined in :py:func:`qiskit.transpiler.PassManager.append`)
-                to be added to the pass manager schedule.
+            passes: A pass set to be added to the pass manager schedule.
             max_iteration: The maximum number of iterations the schedule will be looped if the
                 condition is not met.
         """
-        super().__init__(passes, max_iteration)
-        self.property_set = None
+        super().__init__(max_iteration=max_iteration)
+
+        # This is just for backward compatibility.
+        # RunningPassManager is now almost just an alias of FlowControllerLinear,
+        # but there is slight API difference and backward compatible RunningPassManager object
+        # must come here. Once after the API is aligned, this line will be removed.
+        self._flow_controller = RunningPassManager()
```

Comment:
Similar to previous comment, and the typing here really illustrates my point: this feels very much like an object with immutable "task list" state that `PassManager` wants to create during `run`, not here.